### PR TITLE
Sample Yaml Service and Endpoint

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Closes: #127 
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

File: deploy/sample-gluster-service.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: glusterfs-cluster
+  labels:
+    app: gluster
```
Comment (humblec) [inline deploy/sample-gluster-service.yaml]: May be we dont need this label.

Comment (MohamedAshiqrh) [inline deploy/sample-gluster-service.yaml]: Sure

**Comment by MohamedAshiqrh** [comment]
Tested these files.

File: deploy/sample-gluster-service.yaml (hunk 1 above)
Comment (MohamedAshiqrh) [inline deploy/sample-gluster-service.yaml]: Done

**Comment by humblec** [comment]
LGTM

**Comment by humblec** [comment]
@jarrpa we are keeping this samples for static provisioning examples.

**Comment by jarrpa** [comment]
@humblec If that's the case these files should go under the `docs/examples` directory, as they are not required this deployment.

**Comment by humblec** [comment]
@jarrpa ah.ok. will keep it under docs/examples

**Pull request merged**